src/App/pinJSONAndImage/pin.middleware.ts

```diff
@@ -4,7 +4,7 @@ import { graphqlUploadExpress } from 'graphql-upload'
 @Injectable()
 class PinMiddleware implements NestMiddleware {
   use(req: any, res: any, next: () => void) {
-    graphqlUploadExpress({ maxFileSize: 5000000, maxFiles: 3 })(req, res, next)
+    graphqlUploadExpress({ maxFileSize: 10000000, maxFiles: 3 })(req, res, next)
```

Comment:
```suggestion
    graphqlUploadExpress({ maxFileSize: 2000000, maxFiles: 3 })(req, res, next)
```

in the specs i set 10MB but I dont see any reason to upload a file > 2MB 👍🏻